gateway/src/main/java/io/camunda/zeebe/gateway/Gateway.java

```diff
@@ -235,6 +237,10 @@ private ServerServiceDefinition applyInterceptors(final BindableService service)
     Collections.reverse(interceptors);
     interceptors.add(new ContextInjectingInterceptor(queryApi));
     interceptors.add(MONITORING_SERVER_INTERCEPTOR);
+    if (AuthMode.IDENTITY.equals(gatewayCfg.getSecurity().getAuthentication().getMode())) {
```

Comment:
:upside_down_face: While `equals` on an enum is equivalent to `==`, the `==` operator is inherently null safe (i.e. you don't have to care on which side you put it), so for enums it has a slight edge I think

Reply:
habits :) In the end I think all that matters is alignment here on preferring one way and I'll adopt